src/main/proto/ga4gh/vrsatile/v1/vrsatile.proto

```diff
@@ -37,26 +22,29 @@ message Expression {
 }
 
 message VariationDescriptor {
-//  Variation variation = 1;
-  ValueObjectDescriptor valueObjectDescriptor = 1;
-  // The molecular context of this variant. Must be one of “genomic”, “transcript”, or “protein”.
-  MoleculeContext molecule_context = 2;
-  // The structural variant type associated with this variant. We RECOMMEND a descendent term of SO:0001537.
-  string structural_type = 3;
-  // Typically HGVS or ISCN nomenclature expressions. Other systems relevant to the description of variation MAY be used.
-  repeated Expression expressions = 4;
+  string id = 1;
+  org.ga4gh.vrs.v1.Variation variation = 2;
+  string label = 3;
+  string description = 4;
   // A specific gene context that applies to this variant.
-  // TODO: just make this a GeneDescriptor - easier to handle and more consistent. Can still just represent a curie with a gene descriptor
-  oneof gene_context {
-    string curie = 5;
-    GeneDescriptor gene_descriptor = 6;
-  }
-  // A Sequence corresponding to a “ref allele”, as used in HGVS, SPDI, and VCF variation formats. Useful for validation.
-  string ref_allele_seq = 7;
-  LocationDescriptor location_descriptor = 8;
-  SequenceDescriptor sequence_descriptor = 9;
-  // We RECOMMEND that the allelic_state of variant be described by terms from the Genotype Ontology (GENO). These SHOULD descend from concept GENO:0000875.
-  string allelic_state = 10;
+  GeneDescriptor gene_context = 5;
+  // HGVS, SPDI, and gnomAD-style strings should be represented as Expressions
+  repeated Expression expressions = 6;
+  // Allele registry, ClinVar, or other related IDs should be included as xrefs
+  repeated string xrefs = 7;
```

Comment:
rename `alternate_ids`?

Reply:
This sounds good to me–related to below discussion by @pnrobinson. I agree that a more tightly-constrained field labeled `alternate_ids` makes sense, and we should add to VRSATILE.